### PR TITLE
added more logs for elastic client

### PR DESCRIPTION
**Pull request by VitaliyVoronov**

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -71,6 +74,7 @@ exports.indexall = function (bulk, index) {
             type: index,
             body: bulk
         }, (err) => {
+            logger.error('Elasticsearch indexall error: ', err);
```
Comment (Verbalist) [inline src/utils.js]: ``

Comment (axelsheva) [inline src/utils.js]: Everywhere should be `err.message`

File: src/utils.js [hunk 2]
```diff
@@ -71,6 +74,7 @@ exports.indexall = function (bulk, index) {
             type: index,
             body: bulk
         }, (err) => {
+            logger.error(`[Elasticsearch] [indexall]: `, err);
```
Comment (Verbalist) [inline src/utils.js]: `[Elasticsearch] [indexall]: ${err}`,

File: src/utils.js [hunk 3]
```diff
@@ -1,5 +1,8 @@
 const esClient = require('./elasticsearch/connection');
 const Accounts = require('./modules/accounts');
+const Logger = require('./logger.js');
+
+const logger = new Logger().logger;
```
Comment (axelsheva) [inline src/utils.js]: Can new instance break another logger?

Comment (Verbalist) [reply src/utils.js]: nope singleton

**Pull request merged**